pkgs/tools/misc/mouse-m908/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, libusb1, cmake, udev, pkg-config }:
+
+stdenv.mkDerivation rec {
```

Comment:
you should do and remove the `rec` keyword here.
People prefer to use final attrs.
for more information see: 
    History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
    Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
    Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Feel free to ask questions.